waiter/src/waiter/util/client_tools.clj

```diff
@@ -987,6 +987,18 @@
   [waiter-url]
   (= "composite" (get-authenticator-kind waiter-url)))
 
+(defn using-saml-authenticator?
```

Comment:
Should we make this private so someone doesn't inadvertently use this function?

Reply:
if someone wants to use it, they can

Reply:
the function does what it says